presto-main/src/main/java/com/facebook/presto/operator/aggregation/multimapagg/MultimapAggregationFunction.java

```diff
@@ -149,8 +149,7 @@ public static void output(Type keyType, Type valueType, MultimapAggregationState
 
             state.forEach((key, value, keyValueIndex) -> {
                 // Merge values of the same key into an array
-                if (!keySet.contains(key, keyValueIndex)) {
-                    keySet.add(key, keyValueIndex);
+                if (keySet.add(key, keyValueIndex)) {
```

Comment:
Some linter must have this, and other simiilar optimizations. Might be worth finding one and running it on worker code

Reply:
This is a new function I added at some point when implementing set_agg but didn't bother fixing old code